### PR TITLE
Quirks: Add tests for "html fills viewport" and "body fills html"

### PR DESCRIPTION
**Pull request by andreubotella**
The "`html` fills viewport" quirk wasn't tested at all; and the tests for "`body` fills `html`" only covered the exceptions to the quirk rather than the cases where it should apply. This PR adds the missing coverage.

### CODE REVIEW

File: quirks/html-fills-viewport-quirk-ref.html [hunk 1]
```diff
@@ -0,0 +1,6 @@
+<!DOCTYPE html>
+<body style="width: 100vw; height: 100vw; margin: 0; padding: 10px; box-sizing: border-box">
```
Comment (zcorpan) [inline quirks/html-fills-viewport-quirk-ref.html:2]: The height should be `100vh`

```suggestion
<body style="width: 100vw; height: 100vh; margin: 0; padding: 10px; box-sizing: border-box">
```

**Review by zcorpan — CHANGES_REQUESTED** [review]
Thanks! I found one mistake in the ref.

**Comment by zcorpan** [comment]
Hmm, an aspect that the tests don't exercise well is which margins are being substracted from the block-size, since the margins are the same on all sides. Maybe use different margins on all sides, like `10px 11px 12px 14px`?

**Review by zcorpan — APPROVED** [review]

**Comment by zcorpan** [comment]
Thank you! Are you interested in filing a bug for WebKit?

Test results: https://wpt.fyi/results/quirks?label=pr_head&max-count=1&pr=26438

**Comment by andreubotella** [comment]
> Thank you! Are you interested in filing a bug for WebKit?

https://bugs.webkit.org/show_bug.cgi?id=219018

**Pull request merged**